include/openspace/scripting/scriptengine.h

```diff
@@ -49,15 +49,17 @@ namespace openspace::scripting {
  */
 class ScriptEngine : public Syncable {
 public:
-    using ScriptCallback = std::function<void(ghoul::Dictionary)>;
-    BooleanType(ShouldBeSynchronized);
-    BooleanType(ShouldSendToRemote);
-
-    struct QueueItem {
-        std::string script;
-        ShouldBeSynchronized shouldBeSynchronized;
-        ShouldSendToRemote shouldSendToRemote;
-        ScriptCallback callback;
+    struct Script {
+        BooleanType(ShouldBeSynchronized);
+        BooleanType(ShouldSendToRemote);
+        BooleanType(ShouldBeLogged);
+        using Callback = std::function<void(ghoul::Dictionary)>;
+
+        std::string code;
+        ShouldBeSynchronized synchronized = ShouldBeSynchronized::Yes;
+        ShouldSendToRemote sendToRemote = ShouldSendToRemote::Yes;
```

Comment:
I think it would be good with a comment to explain the difference between remote and synchronized